rest/taskana-rest-spring-example/src/test/java/pro/taskana/doc/api/MonitorControllerRestDocumentation.java

```diff
@@ -77,31 +76,37 @@ public void setUp() {
                 fieldWithPath("sumRow.total").description("Total number of tasks"),
                 fieldWithPath("_links.self.href").ignored()
         };
-        
-        countByStateFieldDescriptors = new FieldDescriptor[] {
-                fieldWithPath("[]..state").description("The state the tasks are in"),
-                fieldWithPath("[]..counter").description("Number of tasks in the corresponding state")
-        };
     }
     
     @Test
     public void getTaskStatusReport() throws Exception {
         this.mockMvc.perform(RestDocumentationRequestBuilders
-                .get("http://127.0.0.1:" + port + "/v1/monitor/taskStatusReport")
+                .get("http://127.0.0.1:" + port + "/v1/monitor/tasks-status-report")
                 .header("Authorization", "Basic YWRtaW46YWRtaW4="))
         .andExpect(MockMvcResultMatchers.status().isOk())
         .andDo(MockMvcRestDocumentation.document("GetTaskStatusReportDocTest",
                 responseFields(taskStatusReportFieldDescriptors)));
     }
     
     @Test
-    public void getCountByState() throws Exception {
+    public void tasksWorkbasketReport() throws Exception {
         this.mockMvc.perform(RestDocumentationRequestBuilders
-                .get("http://127.0.0.1:" + port + "/v1/monitor/countByState?states=READY,CLAIMED,COMPLETED")
+                .get("http://127.0.0.1:" + port + "/v1/monitor/tasks-workbasket-report?daysInPast=4&states=READY,CLAIMED,COMPLETED")
                 .accept("application/hal+json")
-                .header("Authorization", "Basic dGVhbWxlYWRfMTp0ZWFtbGVhZF8x"))
+                .header("Authorization", "Basic YWRtaW46YWRtaW4="))
         .andExpect(MockMvcResultMatchers.status().isOk())
-        .andDo(MockMvcRestDocumentation.document("GetCountByStateDocTest",
```

Comment:
You have removed an old identifier. The link from the `rest-api.adoc` file will no longer work. Therefore the documentation wil lbe displayed wrong. See attached image
![image](https://user-images.githubusercontent.com/15628173/45205743-5ae1af80-b283-11e8-8b7c-83b82f5f9cdd.png)